reduction_scripts/reduce_data.py

```diff
@@ -916,43 +916,99 @@ def run_save_3dcube(metadata, prev_suffix, curr_suffix, **args):
     # INICIATE THE SCRIPT
     # --------------------------------------------
 
-    # Parse command line arguments
-    if len(sys.argv) < 2:
-        print("Error: Please enter a raw data directory.")
-        sys.exit(1)
+    # Initialize ArgumentParser with a description
+    parser = argparse.ArgumentParser(
+        description="The Python data reduction pipeline for WiFeS."
+    )
+
+    # The raw data directory is a required positional argument
+    parser.add_argument("data_dir", type=str, help="Path to the raw data directory.")
+
+    # Option for specifying the path to the red parameters JSON file
+    parser.add_argument(
+        "--red-params",
+        type=str,
+        help="Optional: Path to the configuration JSON file containing parameters for reducing the blue arm.",
+    )
+
+    # Option for specifying the path to the blue parameters JSON file
+    parser.add_argument(
+        "--blue-params",
+        type=str,
+        help="Optional: Path to the configuration JSON file containing parameters for reducing the blue arm.",
+    )
+
+    args = parser.parse_args()
 
-    # Read the raw data directory from command line
-    data_dir = os.path.abspath(sys.argv[1]) + "/"
+    # Validate and process the data_dir
+    data_dir = os.path.abspath(args.data_dir)
+    if not data_dir.endswith("/"):
+        data_dir += "/"
+    print(f"Processing data in directory: {data_dir}")
```

Comment:
Remove the print, this is info just for our checks.

Reply:
I added these print statements to assist users running the pipeline locally, similar to the messages in **Using blue/red parameters from:**. It might be beneficial to include them in the log as well. While this information is pretty obvious in automatic pipeline executions, it could serve as a helpful warning or assist in identifying mistakes (e.g., using an incorrect set of parameters) for local users. What are your thoughts?